PLog.cs

```diff
@@ -1,13 +1,45 @@
-﻿using System;
-
-namespace p2pcopy
-{
-	public class PLog
-	{
-		public static void DEBUG(string fmt, params object[] args)
-		{
-			//Console.WriteLine(string.Format(fmt, args));
-		}
-	}
-}
-	
\ No newline at end of file
+﻿using System;
+
+namespace p2pcopy
+{
+    public class PLog
+    {
+        private static bool debug = false;
+        private static bool verbose = false;
+
+        public static void SetDebug()
+        {
+            debug = true;
+        }
+
+        public static void SetVerbose()

```

Comment:
haha this is cheating, I mean, make the fields private but then add public functions that set them? what you really want is a private property whose setter is internal, and when doing this, make them PascalCase because the fields become properties

Reply:
Done!